packages/config/src/bundleConfig.ts

```diff
@@ -10,10 +10,20 @@ export type TypeScriptValidationOptions = {
   throwOnError?: boolean;
 };
 
+export type CustomSerializationPluginOption = {
+  path: string;
+  options?: object;
+};
+
 /**
  * Parameters controlling bundler plugins.
  */
 export type BundlerPlugins = {
+  /**
+   * Specify additional custom metro serializer plugins
+   */
+  customSerializationPlugins?: CustomSerializationPluginOption[];
```

Comment:
Can we rename this to make it clear that it won't replace the built-in plugins? E.g.:

```suggestion
  extraSerializationPlugins?: ExtraSerializationPluginOption[];
```